### ISSUE TITLE
Distribute package via conda

### ISSUE THREAD

**Issue by nreinicke** [NREL/mappymatch#106]
In addition to distributing the package via PyPi we could also distribute via conda.

**Comment by Rowlando13** [comment]
I agree with this since our primary method to install is via conda. Do we have any particular goals we want to meet before we attempt this? I would recommend ~90 percent docstring coverage and updates to the docs. I think our test coverage is good. Last time I checked it was about ~80 percent.

**Comment by Rowlando13** [comment]
To clarify, I would be happy to take this issue when we are ready.

**Comment by nreinicke** [comment]
yeah I think docs should be the highest priority to encourage users who discover the package to stay. I would be down to hold on this until we have good docs for the package.
